### PR TITLE
Fixes #2667 project conversion tests failing

### LINKED ISSUE #2667: Project conversion tests failing
The message is the same for all

OneTimeSetUp: PKSim.Infrastructure.ORM.Repositories.CalculationMethodNotFoundException : Calculation method 'BMI' was not found.

### PR DESCRIPTION
**Pull request by msevestre**

### CODE REVIEW

File: src/PKSim.Infrastructure/ORM/Mappers/CalculationMethodToCoreCalculationMethodMapper.cs [hunk 1]
```diff
@@ -91,9 +93,7 @@ private DescriptorCriteria descriptorCriteriaFrom(int parameterId)
       {
          var descriptorCriteria = new DescriptorCriteria();
 
-         var conditions = from paramDescr in _flatCalculationMethodParameterDescriptorConditionRepository.All()
-            where paramDescr.ParameterId == parameterId
-            select paramDescr;
+         var conditions = _flatCalculationMethodParameterDescriptorConditionRepository.All().Where(x => x.ParameterId == parameterId);
```
Comment (msevestre) [inline src/PKSim.Infrastructure/ORM/Mappers/CalculationMethodToCoreCalculationMethodMapper.cs:96]: simple code reformatting. Nothing happening here

File: src/PKSim.Infrastructure/PKSim.Infrastructure.csproj [hunk 2]
```diff
@@ -17,18 +17,11 @@
     <NoWarn>1591</NoWarn>
 
   </PropertyGroup>
-
   <ItemGroup>
     <Compile Include="..\..\SolutionInfo.cs" Link="Properties\SolutionInfo.cs" />
   </ItemGroup>
 
   <ItemGroup>
-    <Folder Include="ProjectConverter\v6_0\" />
```
Comment (msevestre) [inline src/PKSim.Infrastructure/PKSim.Infrastructure.csproj:26]: those are empty now. Go away

File: src/PKSim.Infrastructure/ProjectConverter/v12/Converter11To12.cs [hunk 3]
```diff
@@ -23,9 +41,29 @@ public class Converter11To12 : IObjectConverter
...
 
+      private void convertCalculationMethods(XElement calculationMethodCacheElement)
+      {
+         var all = calculationMethodCacheElement.Element("All");
+         foreach (var calculationMethodElement in all.Elements())
+         {
+            var name = calculationMethodElement.GetAttribute("name");
+            if (string.Equals(name, ConverterConstants.CalculationMethod.BMI))
+            {
+               calculationMethodElement.SetAttributeValue("name", ConverterConstants.CalculationMethod.Individual_AgeDependent);
```
Comment (msevestre) [inline src/PKSim.Infrastructure/ProjectConverter/v12/Converter11To12.cs:61]: I have renamed this one method to be everything that is Age Dependent formula

File: src/PKSim.Infrastructure/ProjectConverter/v12/Converter11To12.cs [hunk 4]
```diff
@@ -52,5 +90,47 @@ private void renameSimulationSettings(XElement simulationElement)
...
+         if (!individual.IsHuman)
+            return;
+
+         if (individual.Organism.Parameter(CoreConstants.Parameters.PMA) != null)
+            return;
+
+         var defaultHuman = _defaultIndividualRetriever.DefaultHuman();
+         var pma = defaultHuman.Organism.Parameter(CoreConstants.Parameters.PMA);
+
+         individual.Organism.Add(_cloner.Clone(pma));
```
Comment (msevestre) [inline src/PKSim.Infrastructure/ProjectConverter/v12/Converter11To12.cs:128]: do need to add this after all :)

**Comment by msevestre** [comment]
Going to merge so that I can run the integrations tests

**Review by Yuri05 — COMMENTED** [review]
approved

**Pull request merged**